frontend/cypress/integration/featureFiles/wizard_istio_config.feature

```diff
@@ -51,6 +51,17 @@ Feature: Kiali Istio Config wizard
     And user creates the istio config
     Then the "K8sGateway" "k8sapigateway" should be listed in "bookinfo" namespace
 
+  @gateway-api
+  @bookinfo-app
+  Scenario: Create a K8s Reference Grant scenario
+    When user deletes k8sreferencegrant named "k8srefgrant" and the resource is no longer available
+    And user clicks in the "K8sReferenceGrant" Istio config actions
+    And user sees the "Create K8sReferenceGrant" config wizard
+    And user types "k8srefgrant" in the "name" input
+    And user previews the configuration
```

Comment:
I think you should test `From Namespace`, `From Kind` and `To Kind` selects, although default values are valid for K8sReferenceGrant creation.

There is following action in `wizard_istio_config.feature`:
```
And('user chooses {string} mode from the {string} select', (option: string, id: string) => {
  cy.get(`select[id="${id}"]`).select(option);
});
```
Then it should be easy to add such actions into Reference Grant test scenario